### ISSUE TITLE
Structure of the Model Input Data

### ISSUE THREAD

**Issue by sagedemage** [sagedemage/movie_recommendation_system#16]
The first return value of the `__getitem__` method of the `MovieDataset` class should be a tensor containing the values, **movie_id** and **genre_value**. 

The *generate dataset* script should create a modified dataset with the **genre_value** column filled out.

**Formulas**
The formula to calculate the **individual genre value**:  $$\frac{1}{2}a+2=x$$
- $a =$ label_id 
- $x =$ individual_genre_value

The formula to get the **genre value** for a set of genres: $$y = sum(x)$$
- $y =$ genre_value

**Requirements**
1. The `__getitem__` method of the `MovieDataset` class should return a tensor with two values: 
    1. Tensor[movie_id, genre_value] 
    2. label
2. The modified dataset must have the **genre_value** column filled out via the *generate dataset* script.
3. Follow the formulas to calculate the **genre value**.

You can create a Tensor with this code:
```
tensor = torch.tensor([0, 1])
print(tensor)
```
Output:
```
tensor([0,  1])
```

## Resources
- https://pytorch.org/tutorials/beginner/basics/data_tutorial.html
- https://pytorch.org/vision/master/generated/torchvision.io.read_image.html
- https://pytorch.org/vision/master/generated/torchvision.io.decode_image.html#torchvision.io.decode_image
- https://pytorch.org/docs/stable/generated/torch.tensor.html#torch.tensor

**Comment by sagedemage** [comment]
```
git fetch origin
git checkout 16-structure-of-the-model-input-data
```

**Comment by sagedemage** [comment]
I am not able to implement this feature. I can't use a tensor array of size 2 when training and predicting with the model.
